<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserVolumeDlg</class>
 <widget class="QDialog" name="UserVolumeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volume</string>
  </property>
  <property name="locale">
   <locale language="C" country="AnyCountry"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Voice Volume Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Volume</string>
          </property>
          <property name="buddy">
           <cstring>voicevolSlider</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="voicevolSlider">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Stereo Option</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="voiceleftChkBox">
           <property name="text">
            <string>Mute left</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="voicerightChkBox">
           <property name="text">
            <string>Mute right</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Media File Volume Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Volume</string>
          </property>
          <property name="buddy">
           <cstring>mfvolSlider</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="mfvolSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Stereo Option</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="mfleftChkBox">
           <property name="text">
            <string>Mute left</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="mfrightChkBox">
           <property name="text">
            <string>Mute right</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>voicevolSlider</tabstop>
  <tabstop>voiceleftChkBox</tabstop>
  <tabstop>voicerightChkBox</tabstop>
  <tabstop>mfvolSlider</tabstop>
  <tabstop>mfleftChkBox</tabstop>
  <tabstop>mfrightChkBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UserVolumeDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UserVolumeDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
